pkg/plugin/dropreason/dropreason_linux_test.go

```diff
@@ -101,6 +101,11 @@ func TestCompile(t *testing.T) {
 		t.Fatalf("Expected no error. Error: %+v", err)
 	}
 
+	err = p.Generate(context.Background())
+	if err != nil {
+		t.Fatalf("Expected no error. Error: %+v", err)
```

Comment:
I typically just do this, but nbd either way:

```suggestion
		t.Fatal("Expected no error. Error:", err)
```